examples/gamma_correction/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+set(CMAKE_CXX_COMPILER "dpcpp")
```

Generally I don't like explicit set of `CMAKE_CXX_COMPILER`, but for me it would be ok to add some flexibility:
```suggestion
if (NOT CMAKE_CXX_COMPILER)
    set(CMAKE_CXX_COMPILER "dpcpp")
endif()
```

---

As another way we can remove such `set(CMAKE_CXX_COMPILER "dpcpp")` and mention in the Readme that users should do it explicitly